admin_panel/src/commonMain/kotlin/net/thechance/mena/admin_panel/data/mapper/dateMapper.kt

```diff
@@ -0,0 +1,9 @@
+package net.thechance.mena.admin_panel.data.mapper
+
+import kotlinx.datetime.LocalDate
+
+fun parseLocalDateOrDefault(value: String?) = value?.let {
+    runCatching { LocalDate.parse(it) }.getOrDefault(DEFAULT_DATE)
+} ?: DEFAULT_DATE
```

Comment:
can we make default date a parameter in the function with default value as the DEFAULT_DATE

```suggestion
fun parseLocalDateOrDefault(value: String?, defaultDate: LocalDate = DEFAULT_DATE) = value?.let {
    runCatching { LocalDate.parse(it) }.getOrDefault(defaultDate)
} ?: defaultDate
```